journal.py

```diff
@@ -25,7 +25,9 @@
     pass
 
 path = os.getenv('HOME', os.path.expanduser('~')) + '/.tnote'
-db = SqliteDatabase(path + '/diary.db')
+
+with open('cipher_key.txt', 'r') as cipher_file:
```

Comment:
On first thoughts, I think `cipher_key.txt` should not be visible to anybody. Placing it on the `~/.tnote` folder isn't any good to.

A passphrase approach combined with encryption seems viable. What's your opinion on that?